repos/system_upgrade/el7toel8/actors/removebootfiles/tests/unit_test.py

```diff
@@ -0,0 +1,39 @@
+import os
+import pytest
+
+from leapp.exceptions import StopActorExecution
+from leapp.libraries.actor import library
+from leapp.models import BootContent
+
+
+class remove_file_mocked(object):
+    def __init__(self):
+        self.called = 0
+        self.files_to_remove = []
+
+    def __call__(self, file):
+        self.called += 1
+        self.files_to_remove.append(file)
+
+
+def test_remove_boot_files(monkeypatch):
+    # BootContent message available
+    def consume_message_mocked(*models):
+        yield BootContent(kernel_path='/abc', initram_path='/def')
+    monkeypatch.setattr('leapp.libraries.stdlib.api.consume', consume_message_mocked)
+    monkeypatch.setattr(library, 'remove_file', remove_file_mocked())
+
+    library.remove_boot_files()
+
+    assert library.remove_file.files_to_remove == ['/abc', '/def']
+
+    # No BootContent message available
+    def consume_no_message_mocked(*models):
+        yield None
+    monkeypatch.setattr('leapp.libraries.stdlib.api.consume', consume_no_message_mocked)
+    monkeypatch.setattr(library, 'remove_file', remove_file_mocked())
+
+    with pytest.raises(StopActorExecution):
+        library.remove_boot_files()
```

Comment:
The test is failing on this line. I need to monkeypatch `api.current_logger()` because it returns _NoneType_.